assets/locales/en.json

```diff
@@ -141,5 +141,13 @@
   "runGarbageCollectorDone": {
     "title": "Garbage collector",
     "message": "The garbage collector ran successfully."
+  },
+  "moveRepositoryFailed": {
+    "title": "Move repository",
+    "message": "Could not move the repository from { currDir } to { newDir }."
```

Comment:
Put double quotes around the values of currDir and newDir? (Sorry to not suggest code itself, am on my phone!)